### PR TITLE
* Basic UI work for #1148

### PR DESCRIPTION
**Pull request by Wagnerp**
* Basic UI work for #1148

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs [hunk 1]
```diff
@@ -2250,4 +2250,22 @@ public enum FadeSpeedChoice
...
+    {
+        None = DialogResult.None,
+        Ok = DialogResult.OK,
+        Cancel = DialogResult.Cancel,
+        Abort = DialogResult.Abort,
+        Retry = DialogResult.Retry,
+        Ignore = DialogResult.Ignore,
+        Yes = DialogResult.Yes,
+        No = DialogResult.No,
+        Checked = 8,
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: Move these to higher numbers,
And Changed CheckState to be Intermediate>
Currently only 1 of these states can be returned

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: @Smurf-IV Any numbers to avoid, i.e. can they be 31 & 32?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: It's an integer, so go big ;-)

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: @Smurf-IV Should it be a `uint` as all default values come from [here](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-messagebox)?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: No, As they are Return values, and as such should be `int` according to that link.
But consider this scenario..
- Question is `MB_CANCELTRYCONTINUE` based
- Checkbox is set as an option
- How does it return a Continue with a checkbox intermediate state set ?

In other words, I do not think returning the checkstate is going to work.
And only the `get` CheckState is required as an API !

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs:2266]: There might be another way, just use `Checked` as it's a bool. Checked would be yes & unchecked would be no?

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs]: Used higher numbers (1006 & 1007)

**Comment by Smurf-IV** [comment]
Remove the ProgressBar updates, as they belong to another Request, that is now in Ext ??

**Comment by Wagnerp** [comment]
> Remove the ProgressBar updates, as they belong to another Request, that is now in Ext ??

Ready to go

**Comment by Wagnerp** [comment]
@Smurf-IV This is now ready to merge :)

**Pull request merged**